### PR TITLE
Submitting Clarinet version 0.14.1.yaml

### PR DESCRIPTION
**Pull request by CharlieC3**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21192)

### CODE REVIEW

File: manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+PackageVersion: 0.14.1
+Installers: 
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/hirosystems/clarinet/releases/download/v0.14.1/clarinet-0.14.1-x86_64.msi
+  InstallerSha256: 311162885122C8EE018DAF46EBBE283C6626DBBA4A426627F8AE1732385DBC87
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /passive
```
Comment (ItzLevvie) [inline manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml:11]:
```suggestion
```

Comment (ItzLevvie) [reply manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml:11]: These aren't needed here; the client determines which switches to use based on `InstallerType`.

File: manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: HiroSystems.Clarinet
+PackageVersion: 0.14.1
+PackageLocale: en-US
+Publisher: Hiro Systems
+PublisherUrl: https://www.hiro.so/
+PrivacyUrl: https://www.hiro.so/p/terms-privacy
+PackageName: Clarinet
```
Comment (ItzLevvie) [inline manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.locale.en-US.yaml:7]:
```suggestion
PackageName: clarinet
```
must match Control Panel.

File: manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: HiroSystems.Clarinet
+PackageVersion: 0.14.1
+Installers: 
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/hirosystems/clarinet/releases/download/v0.14.1/clarinet-0.14.1-x86_64.msi
+  InstallerSha256: 311162885122C8EE018DAF46EBBE283C6626DBBA4A426627F8AE1732385DBC87
```
Comment (ItzLevvie) [inline manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml:8]:
```suggestion
  InstallerSha256: 311162885122C8EE018DAF46EBBE283C6626DBBA4A426627F8AE1732385DBC87
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by CharlieC3** [comment]
Suggestions implemented, thanks for your feedback @ItzLevvie

File: manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml (hunk 3 above)
Comment (CharlieC3) [reply manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml:8]: Implemented, but Github isn't auto-resolving this suggestion.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**